scripts/lib.sh

```diff
@@ -169,13 +198,36 @@ function start_lnd() {
   echo $! >> $FM_PID_FILE
 }
 
-function start_gatewayd() {
-  echo "starting gatewayd"
-  await_gateway_cln_extension
-  await_fedimint_block_sync
-  $FM_BIN_DIR/fixtures gatewayd &
-  gw_connect_fed
-  echo "started gatewayd"
+function start_cln_gateway() {
+  await_server_on_port 8177 # grpc port
+
+  export FM_GATEWAY_DATA_DIR=$FM_TEST_DIR/gw-cln
+  export FM_GATEWAY_LISTEN_ADDR="127.0.0.1:8175"
+  export FM_GATEWAY_API_ADDR="http://127.0.0.1:8175"
+  $FM_BIN_DIR/fixtures gatewayd cln &
+  echo $! >> $FM_PID_FILE
+  echo "started cln gateway"
+
+  connect_cln_gateway
+}
+
+function start_lnd_gateway() {
+  await_server_on_port 11009
+
+  # start lnd gw
+  export FM_GATEWAY_DATA_DIR=$FM_TEST_DIR/gw-lnd
+  export FM_GATEWAY_LISTEN_ADDR="127.0.0.1:28175"
+  export FM_GATEWAY_API_ADDR="http://127.0.0.1:28175"
+  $FM_BIN_DIR/fixtures gatewayd lnd &
+  echo $! >> $FM_PID_FILE
+  echo "started lnd gateway"
+
+  connect_lnd_gateway
+}
```

Comment:
Does it make sense to have these env variables to be passed as arguments by rust? [This](https://github.com/dhruv-1001/fedimint/blob/tmux-test/fedimint-testing/src/bin/fixtures.rs#L397-L416) is what I was working on for this issue.

Reply:
Yea I think that looks nice. The only problem is that https://github.com/fedimint/fedimint/pull/1910 is a huge refactor in this direction. But still a good change that we can port over once #1910 lands.